src/lib/create-config.ts

```diff
@@ -232,10 +232,14 @@ const formatServerOptions = (
     if (!serverOptions) return;
 
     /* eslint-disable-next-line */
-    return {
-        ...serverOptions,
-        baseUriPath: formatBaseUri(serverOptions.baseUriPath),
-    };
+    if(serverOptions.baseUriPath) {
+        return {
+            ...serverOptions,
+            baseUriPath: formatBaseUri(serverOptions.baseUriPath),
+        };
+    } else {
+        return serverOptions;
+    }
```

Comment:
Could this potentially be rewritten like this or wouldn't that work? Might need to edit it if we want `''` as a valid string.

```suggestion
    return {
        ...serverOptions,
        baseUriPath: serverOptions.baseUriPath ? formatBaseUri(serverOptions.baseUriPath) : undefined,
    };
```